tests/test_download.py

```diff
@@ -30,7 +34,7 @@
     CommonCrawlWARCExtractor,
     CommonCrawlWARCIterator,
     JusTextExtractor,
-    ResiliparseExtractor,
+    ResiliparseExtractor,  # noqa: F811
```

Comment:
Why is this `ResiliparseExtractor`? I believe it can be removed (we don't need the line neither noqa)

Reply:
Ah, didn't realize the imports were a bit wonky here. Will fix in a few minutes, just want to wait for tests to run first.